apps/condo/index.js

```diff
@@ -105,6 +105,13 @@ const checks = [
             return SBER_ID_CONFIG.cert
         },
     }),
+    getPfxCertificateHealthCheck({
+        certificateName: 'sber_id_client',
+        getPfxParams: () => {
+            const SBER_ID_CONFIG = conf['SBER_ID_CONFIG'] && JSON.parse(conf['SBER_ID_CONFIG']) || {}
+            return { pfx: SBER_ID_CONFIG.certificate, passphrase: SBER_ID_CONFIG.passphrase }
+        },
+    }),
     getPfxCertificateHealthCheck({
         certificateName: 'sbbol_client',
         getPfxParams: () => {
```

Comment:
You just duplicate the name of healthcheck function. What did you want to do here?

Reply:
Oh, my bad. It not the declaration but it is an usage) Anyway I can't find branch for non pfx format here